bundles/binding/org.openhab.binding.zwave/database/products.xml

```diff
@@ -1705,6 +1705,15 @@
             <Label lang="en">Roller Shutter</Label>
             <ConfigFile>fibaro/fgrm222.xml</ConfigFile>
         </Product>
+		<Product>
```

Comment:
Will do. Having said that, indentation shows aligned in my editor. What tab length do you use, or do you indent using spaces only?